cpp/src/fil/infer.cu

```diff
@@ -504,6 +488,41 @@ void infer_k_launcher(storage_type forest, predict_params params,
   CUDA_CHECK(cudaPeekAtLastError());
 }
 
+template <leaf_algo_t leaf_algo, typename storage_type>
+void infer_k_launcher(storage_type forest, predict_params params,
+                      cudaStream_t stream, int blockdim_x) {
+  // TODO(levsnv): move this to forest init()
```

Comment:
Would it be possible to have it in the forest initialization in this pull request?